src/data_provider/src/osinfo/sysOsParsers.cpp

```diff
@@ -137,15 +137,30 @@ static bool findVersionInStream(std::istream& in,
 bool UnixOsParser::parseFile(std::istream& in, nlohmann::json& info)
 {
     constexpr auto SEPARATOR{'='};
-    static const std::map<std::string, std::string> KEY_MAP
+    bool ret = false;
+
     {
-        {"NAME",             "os_name"},
-        {"VERSION",          "os_version"},
-        {"ID",               "os_platform"},
-        {"BUILD_ID",         "os_build"},
-        {"VERSION_CODENAME", "os_codename"}
-    };
-    const auto ret {parseUnixFile(KEY_MAP, SEPARATOR, in, info)};
+        static const std::map<std::string, std::string> KEY_MAP
+        {
+            {"NAME",             "os_name"},
+            {"VERSION",          "os_version"},
+            {"ID",               "os_platform"},
+            {"BUILD_ID",         "os_build"},
+            {"VERSION_CODENAME", "os_codename"}
+        };
+        ret |= parseUnixFile(KEY_MAP, SEPARATOR, in, info);
```

Comment:
Unnecessary OR-EQUAL operator assignment.

Reply:
I will fix it.